extensions/tags/extend.php

```diff
@@ -133,4 +134,7 @@
 
     (new Extend\SimpleFlarumSearch(DiscussionSearcher::class))
         ->addGambit(TagFilterGambit::class),
+
+    (new Extend\ModelUrl(Tag::class))
+        ->addSlugDriver('default', Utf8SlugDriver::class),
```

Comment:
Should we ship a transliterated driver out of the box as well, to be consistent with other models? And should that be the default?

We should also think a bit about slugging and the admin UI. Should the slug driver interface define a regex for what is / isn't valid? We don't currently stop users from entering a UTF-8 slug in the tag create menu (even though that breaks things), and when this new driver is enabled, we don't stop auto-transliterating slugs.

Reply:
this is the part where I'm conflicted tbh. We have always allowed UTF-8 characters in slugs, but the frontend only takes ASCII characters from the slug title (no transliteration involved).

So I don't think we necessarily need to add a transliterated slug driver, but it would be nice to have. And I don't know if validating the slug (frontend/backend) based on the current driver matters either as no matter what the slug is on the database, the slug driver can change it dynamically based on title.

Reply:
True, I think we adapt it to that it only replaces non-word characters with dashes.